Lib/asyncio/mixins.py

```diff
@@ -0,0 +1,23 @@
+"""Event loop mixins."""
+
+__all__ = ('LoopBoundedMixin',)
+
+import threading
+from . import events
+
+global_lock = threading.Lock()
```

Comment:
```suggestion
_global_lock = threading.Lock()
```
Let's add the underscore prefix to emphasize that the lock is not a part of public API.

Reply:
My bad, fixed